### PR TITLE
feat: added catalyst features in cloud providers list

### PR DESCRIPTION
**Pull request by muthaiyanmani**
## Requirements

 * [x] This is Software as a Service not self hosted
 * [x] It has a free tier not just a free trial
 * [x] Pricing information is clearly visible without signup or phone calls
 * [x] The submission mentions what is free
 * [x] The submission is not already present in the list
 * [x] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,18 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
 
 ## Major Cloud Providers
 
+   * [Catalyst by Zoho](https://catalyst.zoho.com?src=free-for-dev)
```
[the remark this answers is not in the document]
Comment (muthaiyanmani) [reply README.md]: > It's a trial isnt it? Not a free tier.

Its a free tier by monthly.

"Catalyst by Zoho offers a free tier that includes more than 70 components. The free tier is available for an unlimited period of time and is applied at an account level each month. The free tier limit applies across all projects in an account."

> Also we do not accept tracking links like your src here.

Have removed the src params

> We also do not list entries more than once, zoho is already listed this should be added there if it complies.

Catalyst is fully focussing on pro code, severless and development platform oriented and it is not just like other zoho products and this one is completely different and it just backed by zoho.

[the remark this answers is not in the document]
Comment (ripienaar) [reply README.md]: Ah sorry I saw the free trial blurp on the landing and didnt look further, indeed there's a good free tier.

AWS, github, oracle, azure, google etc all have a number of extremely widely varying services.  We still have only one entry each.

[the remark this answers is not in the document]
Comment (muthaiyanmani) [reply README.md]: We offer a wide range of services and components, including related to AI/ML, remote browsers, testing tools, etc. We're also working on some exciting new services internally and will update here upon their release.

[the remark this answers is not in the document]
Comment (ripienaar) [reply README.md]: OK cool, adding your entry is optional.. I'll go ahead and close the PR.

thank yoyu

[the remark this answers is not in the document]
Comment (muthaiyanmani) [reply README.md]: Has this not been added?

[the remark this answers is not in the document]
Comment (muthaiyanmani) [reply README.md]: Cool, Thank you @ripienaar 😀.

Could you provide some feedback? What are we missing?

[the remark this answers is not in the document]
Comment (ripienaar) [reply README.md]: I've repeatedly asked you to put it in the existing zoho offering.  You keep telling how special you are and somehow deserve your own category.

As if your offering is somehow more special than say AWS or GCP or any of the other major players who are all happy being in one entry.

Please do not re-open this PR.

**Pull request closed without merging**